File: 1_iniciante/R/1001.r
#' @source   Beecrowd
#' @problem  1001 - Extremely Basic
#' @subject  Sequential
#'
#' @author   Letícia Cruz
#' @date     2025-08-12

# recebe entrada
input <- file('stdin', 'r')

# tranformação da entrada para inteiro
a <- as.integer(readLines(input, n=1))
b <- as.integer(readLines(input, n=1))

soma = a + b

# paste concatena "X =" com soma e adiciona um espaço entre eles: X = valor
writeLines(paste("X =", soma))

File: 1_iniciante/javascript/1001.js
/**
 * @source Beecrowd
 * @problem 1001 - Extremely Basic
 * @subject Sequential
 *
 * @author Letícia Cruz
 * @date 2020-03-16
 */

var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

var a = parseInt(lines.shift());
var b = parseInt(lines.shift());

console.log("X = " + (a + b));
